scss/components/_commercial.scss

```diff
@@ -1,28 +1,129 @@
+@import "./src/fonts";
+@import "./mixins/flex-mix.scss";
+
 .commercial-data {
-  padding: 25px 0;
   width: $container-width;
   border: 1px solid black;
   background: $grey;
+  padding-bottom: 35px;
 }
 .list-of-data {
   color: $white;
   height: 100%;
   padding-bottom: 215px;
 }
-.list-of-data li {
-  padding: 0 75px;
+.list-of-data-li {
+  width: 25%;
   float: left;
+  margin-top: 60px;
 }
-.list-letters {
+.list-of-data-li-special {
+  width: 18%;
+  float: left;
+  margin-top: 60px;
+}
+%list-letters {
   font-size: 25px;
   align-self: flex-end;
-  margin-left: -6vw;
-  margin-top: 120px;
+  margin: 29px 53px 0 0;
+}
+
+.list-letters {
+  @extend %list-letters;
+}
+
+.list-letters-two {
```

Comment:
Use common class `list-letters` and modificators

Reply:
Done